ppr-ui/src/components/mhrTransfers/LocationChange.vue

```diff
@@ -14,16 +14,70 @@
           itemValue="type"
           variant="filled"
           label="Location Change Type"
+          color="primary"
         />
       </template>
     </FormCard>
 
     <div
       v-if="state.isTransportPermitType"
       id="transport-permit-location-type"
-      class="pt-7"
     >
-      [Transport Permit placeholder]
+      <section
+        id="transport-permit-home-location-type"
+        class="mt-10"
+      >
+        <h2>1. Location Type</h2>
+        <p class="mt-2 mb-6">
+          Enter the new location type of the home.
+        </p>
+
+        <HomeLocationType
+          :validate="false"
+          :class="{ 'border-error-left': false }"
+        />
+      </section>
+
+      <section
+        id="transport-permit-home-civic-address"
+        class="mt-10"
+      >
+        <h2>2. New Civic Address of the Home</h2>
+        <p class="mt-2">
+          Enter the Street Address (Number and Name) and City of new location of the home.
+          Street Address must be entered if there is one.
+        </p>
+        <p class="mt-2">
+          <b>Note:</b> If this manufactured home is being moved to a location outside of B.C.,
+          the status of the home will be exempt upon filing.
+        </p>
+
+        <HomeCivicAddress
+          :schema="CivicAddressSchema"
+          :validate="false"
+          :class="{ 'border-error-left': false }"
+          @isValid="() => {}"
```

Comment:
Is this for part 2?

Reply:
Yea, validations are in the other ticket.